kvbc/test/sparse_merkle_storage/db_adapter_unit_test.cpp

```diff
@@ -184,7 +195,7 @@ TEST_P(db_adapter_custom_blockchain, add_block_return) {
   }
 }
 
-TEST_P(db_adapter_custom_blockchain, add_block_with_non_provable_keys) {
+TEST_P(db_adapter_custom_blockchain, add_block_with_provable_and_non_provable_keys) {
```

Comment:
Nit: I think it will be clearer if we use different naming from `defaultSliver` to designate provable keys and values, in general.

Reply:
Renamed the keys, changed values for non-provables.